journal/api/src/controllers/people.ts

```diff
@@ -64,6 +64,7 @@ export const peopleController = new Elysia()
       return 'Cannot update this person =('
     }
 
+    set.status = 'OK'
```

Comment:
Interesting. Is this not the default?

Reply:
Copilot added it to mine, so I added it to yours. Removing.